### PR TITLE
DQA-012

### PR DESCRIPTION
**Pull request by beshketo**
Added 4 test for Book Store Page

### CODE REVIEW

File: bookstore-application/src/test/java/parallel/BookStoreTest.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    @Given("Open Book Store page")
+    public void openBookStorePage() {
+        open("https://demoqa.com/books");
+    }
+
+    @When("Fill out search field with part of title: {string}")
+    public void FillOutTitleInSearchField(String partTitle) {
+        System.out.println("partTitle: " + partTitle);
```
Comment (DzigMS) [inline bookstore-application/src/test/java/parallel/BookStoreTest.java]: Delete print to console from code.
I guess it's for debugging only.

File: bookstore-application/src/test/java/parallel/BookStoreTest.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        Configuration.timeout = 1000;
+        $("#searchBox").setValue(partTitle);
+    }
+    @When("Fill out the search field with name of author: {string}")
+    public void FillOutAuthorInSearchField(String author) {
+        System.out.println("author: " + author);
```
Comment (DzigMS) [inline bookstore-application/src/test/java/parallel/BookStoreTest.java]: The same...

File: bookstore-application/src/test/java/parallel/BookStoreTest.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    }
+
+    @When("Click on the Login button on Book Store Page")
+    public void clickOnLoginButton() {
+        $("#login").click();
+        Configuration.timeout = 1000;
+    }
+    @When("Fill out all fields for login: {string}, {string}")
+    public void FillOutExistingUserData(String userName, String password) {
+        System.out.println("userName: " + userName);
```
Comment (DzigMS) [inline bookstore-application/src/test/java/parallel/BookStoreTest.java]: And again...

**Review by DzigMS — CHANGES_REQUESTED** [review]

**Review by beshketo — COMMENTED** [review]
i did changes

**Pull request merged**